site/en/blog/full-a11y-tree/index.md

```diff
@@ -0,0 +1,167 @@
+---
+title: 'Full accessibility tree in Chrome DevTools'
+description: >
+  In this blog post we present the new full-page accessibility tree in DevTools and how we designed and implemented it
+authors:
+  - jobay
+layout: 'layouts/blog-post.njk'
+date: 2021-12-13
+hero: 'image/dPDCek3EhZgLQPGtEG3y0fTn4v82/bm9LfyyNqKQdikPW5CjI.jpg'
+alt: ''
+tags:
+  - devtools-engineering
+  - devtools
+---
+
+{% include 'partials/devtools/en/banner.md' %}
+
+Chrome DevTools is launching a full accessibility tree making it easier for developers to get an overview of the whole tree. In this post find out how this tree is created and how to use it in your work.
+
+{% Img src="image/gGNVLl64MBQvhaYBXAAuRiOs3r92/uuDsB0fUMzarbKmRi5Uz.gif", alt="animated screenshot of new accessibility tree", width="800", height="519" %}
```

Comment:
Suggest to upload video instead as it's smaller and better in performance. It will look like gif.

```
{% Video src="video/some-link.mp4", autoplay="true", muted="false", loop="true",  class="screenshot" %}
```

Reply:
```suggestion
{% Video src="video/gGNVLl64MBQvhaYBXAAuRiOs3r92/XQNBCFrPG8YqNSOloDE8.mp4", autoplay="true", muted="false", loop="true", class="screenshot" %}
```